taperole/roles/wheneverize/tasks/main.yml

```diff
@@ -0,0 +1,3 @@
+- name: Wheneverize
```

Comment:
Would be a good idea I think https://github.com/smashingboxes/taperole/issues/76